tools/deployment/package/docker-compose.yaml

```diff
@@ -0,0 +1,127 @@
+name: "clp-package"
+
+include: ["docker-compose.base.yaml"]
+
+# The `x-*` definitions below are duplicated from docker-compose.base.yaml. Refer to that file for
+# documentation.
+x-service-defaults: &service_defaults
+  image: "${CLP_PACKAGE_CONTAINER_IMAGE_REF:-clp-package}"
+  logging:
+    driver: "local"
+  stop_grace_period: "60s"
+  user: "${CLP_FIRST_PARTY_SERVICE_UID_GID:-1000:1000}"
+x-healthcheck-defaults: &healthcheck_defaults
+  interval: "30s"
+  retries: 3
+  start_interval: "2s"
+  start_period: "60s"
+  timeout: "2s"
+x-volume-definitions:
+  clp-config-readonly: &volume_clp_config_readonly
+    type: "bind"
+    source: "${CLP_LOGS_DIR_HOST:-./var/log}/.clp-config.yml"
+    target: "/etc/clp-config.yml"
+    read_only: true
+  clp-logs: &volume_clp_logs
+    type: "bind"
+    source: "${CLP_LOGS_DIR_HOST:-./var/log}"
+    target: "/var/log"
+
+services:
+  query-scheduler:
+    <<: *service_defaults
+    hostname: "query_scheduler"
+    stop_grace_period: "10s"
+    environment:
+      BROKER_URL: "amqp://${CLP_QUEUE_USER:?Please set a value.}\
+        :${CLP_QUEUE_PASS:?Please set a value.}@queue:5672"
+      CLP_DB_PASS: "${CLP_DB_PASS:?Please set a value.}"
+      CLP_DB_USER: "${CLP_DB_USER:?Please set a value.}"
+      CLP_LOGGING_LEVEL: "${CLP_QUERY_SCHEDULER_LOGGING_LEVEL:-INFO}"
+      CLP_LOGS_DIR: "/var/log"
```

Comment:
the PR wrongly changes the query scheduler's logs directory base to `/var/log` (it should be `/var/log/query_scheduler`). a similar issue is found in the compression scheduler.

@coderabbitai create an issue to track the fix